src/lib.rs

```diff
@@ -627,15 +627,21 @@ impl<'v> StarlarkValue<'v> for PythonCallableValue {
         eval: &mut starlark::eval::Evaluator<'v, '_>,
     ) -> anyhow::Result<Value<'v>> {
         Python::with_gil(|py| -> anyhow::Result<Value<'v>> {
-            args.no_named_args()?;
+            // Handle positional arguments.
             let py_args: Vec<PyObject> = (args
                 .positions(eval.heap())?
                 .map(|v| -> PyResult<PyObject> { value_to_pyobject(v) }))
-            .collect::<PyResult<Vec<PyObject>>>()?;
-            convert_to_anyhow(pyobject_to_value(
-                self.callable.call1(py, PyTuple::new_bound(py, py_args))?,
-                eval.heap(),
-            ))
+                .collect::<PyResult<Vec<PyObject>>>()?;
+            
+            // Handle named arguments.
+            let py_kwargs = PyDict::new(py);
```

Comment:
I could not understand why `new_bound` does not work.

Reply:
`new_bound` is for a newer version of pyo3 than this package was on. It's there now.